src/pybind/mgr/mgr_util.py

```diff
@@ -12,7 +12,7 @@
 import logging
 import sys
 from threading import Lock, Condition, Event
-from typing import no_type_check
+from typing import no_type_check, ContextManager
```

Comment:
nit, better off importing more typing types at line 22.

Reply:
thanks for catching that, fixed